cmd/worker/main.go

```diff
@@ -346,6 +346,14 @@ func main() {
 
```

Comment:
I think the CI issue is due to `plz tidy` had not been run after the code changes and thus the `cadence-aws-sdk/aws` package which is a new dependency of the cluster setup package has not been added to the package's BUILD.plz file.